howso/client/base.py

```diff
@@ -3139,6 +3139,117 @@ def _react_series(self, trainee_id: str, params: dict):
 
         return ret, in_size, out_size
 
+    def react_series_stationary(
+        self,
+        trainee_id: str,
+        action_features: Collection[str],
+        *,
+        context_features: t.Optional[Collection[str]] = None,
+        desired_conviction: t.Optional[float] = None,
+        input_is_substituted: bool = False,
+        series_context_features: t.Optional[Collection[str]] = None,
+        series_context_values: t.Optional[TabularData3D] = None,
```

Comment:
We are missing proper serialization and batching of this data. These are not pass through values.